src/AstGen.zig

```diff
@@ -8976,7 +8975,10 @@ fn callExpr(
         } });
     }
 
-    assert(callee != .none);
+    assert(switch (callee) {
+        .direct => |obj| obj != .none,
+        .field => |field| field.obj_ptr != .none,
+    });
```

Comment:
This would be more useful with 2 asserts inside each switch prong, so that if an assertion trips it points to the one that failed